Makefile

```diff
@@ -192,7 +192,7 @@ $(foreach pt,$(PKG_PROFILES),$(eval $(call gen-pkg-target,$(pt))))
 run: $(PROFILE) quickrun
 
 .PHONY: quickrun
-quickrun:
+quickrun: default
```

Comment:
sorry, missed this.
it's not `quick` when it has dependency.

with dependency, it's essentially the same as `make run`

Reply:
Thanks, I overlooked `make run`.